packages/longhorn/package.yaml

```diff
@@ -1,8 +1,8 @@
 url: https://github.com/longhorn/charts.git
 subdirectory: charts/longhorn
-commit: 299f4ab8ad01137c463574fe013da30efc6b05d0
-packageVersion: 01
-releaseCandidateVersion: 01
+commit: 93f250e4a175044d34c6982ea6168b213caeee9f
+packageVersion: 00
+releaseCandidateVersion: 00
```

Comment:
This should be 01

Reply:
Just to clarify @cbron , what does this `releaseCandidateVersion` mean? Also, does it apply to `packageVersion` too?

Reply:
Package version of 00 will turn 1.1.1.1 into 1.1.1.100 which is correct. 
releaseCandidateVersion of 01 will turn 1.1.1.100 into 1.1.1.100-rc01, which is also correct

Reply:
This is the formal release, so it shouldn't be `rc01`. so the original PR looks good?

Reply:
changed.

Reply:
For right now everything in feature charts (this branch) has an rc until it goes live. That transition happens automatically on rancher release. That process will be going away in the future.